src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -1,13 +1,73 @@
 const InterfacePrototype = require('./InterfacePrototype.js')
 const {wkSend, wkSendAndWait} = require('../appleDeviceUtils/appleDeviceUtils')
 const {
-    isApp, notifyWebApp,
+    isApp,
+    notifyWebApp,
+    isTopFrame,
     isDDGDomain,
     formatDuckAddress
 } = require('../autofill-utils')
 const {scanForInputs, forms} = require('../scanForInputs.js')
 
 class AppleDeviceInterface extends InterfacePrototype {
+    constructor () {
+        super()
+        if (isTopFrame) {
+            this.stripCredentials = false
+            window.addEventListener('mouseMove', this)
+        } else {
+            // This is always added as a child frame needs to be informed of a parent frame scroll
+            window.addEventListener('scroll', this)
+        }
+    }
+
+    async listenForSelectedCredential () {
+        const response = await wkSendAndWait('getSelectedCredentials')
+        switch (response.type) {
+        case 'none':
+            // Parent hasn't got a selected credential yet
+            return setTimeout(() => {
+                this.listenForSelectedCredential()
+            }, 100)
+        case 'ok':
+            return this.inboundCredential({
+                detail: {
+                    data: response.data,
+                    configType: response.configType
+                }
+            })
+        case 'stop':
+            // Parent wants us to stop polling
```

Comment:
It's sent whenever the top context parent has a different hostname sender.
In case the user has managed to open two it tells it to stop sending events to the previous popup.